docs/hub/hf-mcp-server.md

```diff
@@ -0,0 +1,61 @@
+# Hugging Face MCP Server
+
+The Hugging Face MCP (Model Context Protocol) Server connects your MCP‑compatible AI assistant (for example Codex, Cursor, VS Code extensions, Zed, or Claude Desktop) directly to the Hugging Face Hub. Once connected, your assistant can search and explore Hub resources and use community tools, all from within your editor or chat.
```

Comment:
I don't see Zed in the settings right now.

Reply:
you can still use it via `others` and it's a seemingly popular IDE: https://zed.dev/docs/ai/mcp